### ISSUE TITLE
Loss slowly decreases

### ISSUE THREAD

**Issue by nikky4D** [mlfoundations/open_clip#41]
Hi, I am attempting to use open_clip for remote images on xview images. I've finding that in the first 2-3 epochs the loss decreases from 3.5 to 2.7 and stays around 2.7 for lr of 8e-6 (see training below). Would anyone have ideas on how I can motivate the learning?

![image](https://user-images.githubusercontent.com/7451106/154493762-e2953b16-9e5a-432f-b12f-ad649335fe03.png)

**Some background on xview:**
My images are derived from xview which is an object detection dataset with images like this: 

![img_nobox_5_0_300](https://user-images.githubusercontent.com/7451106/154494556-81681c6b-07a9-45d3-aded-3811deec9bf7.png)

To generate captions for xview, for each image, I make a single caption for a single bounding box. Hence the same image may be several different captions for that image. Each caption is valid as there may be multiple objects in the image.

**Comment by mitchellnw** [comment]
Any reason for using LR 8e-6? I would go much higher, like 1e-3 or 5e-4.

**Comment by mitchellnw** [comment]
(Also I don't think val loss is a good metric to look at as we've found it's much better to look at zero-shot accuracy)

**Comment by nikky4D** [comment]
Thank you for the response. I have tried 1e-3, and 1e-4. Both have higher losses, and for 1e-3 returns nans for the losses after 2 epochs. I kept decreasing my lr until I hit the stable point.

For zero-shot accuracy metric, do I use a portion of my dataset (my validation set) or do I use a separate dataset for that? Currently, it is turned off as I don't expect the model to do well on imagenet.

**Comment by mitchellnw** [comment]
If you are able to turn your problem into a classification problem I would use your dataset for that. By the way you may run into problems creating a caption from every bbox, as the cars there for example are really small and I would not call that image "a photo of a car". What about making a single caption like "An arial image of 8 buildings and 23 cars"?

**Comment by nikky4D** [comment]
Thanks. I'll try your suggestions. Hopefully loss will behave better with caption change. It will be interesting to see how clip handles it.
